crates/sdk/src/api/method/create.rs

```diff
@@ -60,10 +59,13 @@ macro_rules! into_future {
 			} = self;
 			Box::pin(async move {
 				let router = client.inner.router.extract()?;
-				let cmd = Command::Create {
+
+				let what = resource?;
+
+				let cmd = Command::RawQuery {
 					txn,
-					what: resource?,
-					data: None,
+					query: Cow::Owned(format!("CREATE {}", what.to_sql())),
```

Comment:
Another formatting issue here. Maybe review all the places where this PR constructs a surrealql string.